lib/ansible/executor/play_iterator.py

```diff
@@ -181,6 +185,8 @@ def __init__(self, inventory, play, play_context, variable_manager, all_vars, st
         }
         if gather_timeout:
             setup_task.args['gather_timeout'] = gather_timeout
+        if fact_path:
+            setup_task.args['fact_path'] = fact_path
```

Comment:
Ah okay, I get your point now. That totally makes sense, but would make it inconsistent with the non-inherited defaults for `gather_subset` and `gather_timeout`